src/main/java/seedu/address/logic/Messages.java

```diff
@@ -32,6 +34,13 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
         return MESSAGE_DUPLICATE_FIELDS + String.join(" ", duplicateFields);
     }
 
+    /**
+     * Returns an error message indicating multiple student IDs were provided.
+     */
+    public static String getErrorMessageForDuplicateID() {
```

Comment:
Good addition of specific error messages for duplicate student IDs, this will improve user feedback.